base/compiler/ssair/ir.jl

```diff
@@ -28,12 +28,18 @@ function cfg_delete_edge!(cfg::CFG, from::Int, to::Int)
     nothing
 end
 
+function bb_ordering()
+    lt=(<=)
+    by=x->first(x.stmts)
+    ord(lt, by, nothing, Forward)
+end
```

Comment:
How about making this constant?
```suggestion
const BB_ORDERING = ord((<=), x->first(x.stmts), nothing, Forward)
```